### PR TITLE
Add New Plugin: MoneyDoorPlugin2

### PR DESCRIPTION
**Pull request by HsiaoHanChen**

### CODE REVIEW

**Comment by HsiaoHanChen** [comment]
Merge this request

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin2.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
+
+public class MoneyDoorPlugin2 implements GamePlugin {
+
+    private int[] arr = {-3, -2, -1, 0, 1, 2, 3, 4, 5, 6};
+    private int ONE = 1;
```
Comment (yichenl2) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin2.java:12]: reviewed by yichen . lalalla

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin2.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
```
Comment (yichenl2) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin2.java:5]: 123123

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin2.java (hunk 1 above)
Comment (HsiaoHanChen) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin2.java:12]: hahaha

**Pull request merged**